custom/icds_reports/migrations/0150_update_agg_awc_monthly_view.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 1.11.16 on 2019-10-10
+
+from django.db import migrations
+
+from corehq.sql_db.operations import RawSQLMigration
+
+
+migrator = RawSQLMigration(('custom', 'icds_reports', 'migrations', 'sql_templates', 'database_views'))
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('icds_reports', '0149_auto_20191115_1142'),
+    ]
+
+    operations = [migrator.get_migration('agg_awc_monthly.sql')]
```

Comment:
Does this migration work locally? Asking because the would trigger deleted the view and recreating and not sure whether a view getting deleted works when the view is used in another view creation.

Reply:
> A few questions. Also, did you tested this offlined that you are not able to view these changes in any place when feature flag is off?

yes tested with on/off and working as expected